modules/tensor_mechanics/src/kernels/GeneralizedPlaneStrainOffDiag.C

```diff
@@ -16,6 +16,7 @@
 #include "MooseVariableScalar.h"
 #include "RankTwoTensor.h"
 #include "RankFourTensor.h"
+#include "UserObject.h"
```

Comment:
revert this change

Reply:
@vanwdani, Don't we need this header to define the `UserObject` type in order to call `getUserObject<>` (not `_fe_problem.getUserObject`)? `UserObjectInterface.h` might not be enough for non-unity builds.